GrandCentralBoard/Widgets/Harvest/HarvestWidget.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  HarvestWidget.swift
+//  GrandCentralBoard
+//
+//  Created by Karol Kozub on 2016-04-11.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import GrandCentralBoardCore
+
+
+final class HarvestWidget: Widget {
```

Comment:
It's about tracked time. Widget should be called accordingly. 

 `TrackedTimeStatistics` ?